### DISCUSSION TITLE
Proper way to use this chart for data collection and aggregation

### DISCUSSION THREAD

**Discussion by AntonOfTheWoods** [vectordotdev/helm-charts#383] [Q&A] [answered]
This is a somewhat n00b question but it is unclear how this is supposed to work if you want to both have an agent and an aggregator in your cluster. Is the intended way to support this to install TWO chart instances, one for the agent and one for the aggregator? If this is the case, is there a discussion somewhere why this choice was made, rather than just having both roles potentially managed by a single instance (which would be the least-surprise way of doing it)? Thanks!

**Comment by jszwedko** [comment, marked as answer]
Hi @AntonOfTheWoods !

That's correct, if you want both the "agent" and the "aggregator" the typical pattern is to deploy the chart twice. I do see what you are saying, though, that it could be nice to have the chart deploy both when configured to.
